.travis.yml

```diff
@@ -8,3 +8,4 @@ python: '3.5'
 install: pip install tox
 
 script: tox
+
```

Comment:
you don't need to make dummy changes to trigger travis, you can do `git commit --amend` with no changes which rebuilds the commit, then `git push --force`